tests/api/controllers/robot_multiple_step_advanced/.gitignore

```diff
@@ -0,0 +1 @@
+/robot_multiple_step_advanced
```

Comment:
I would rename this test with "parallel" instead of "advanced". Also, I am not sure we want to test multiple steps here, may only one is sufficient, e.g., robot_parallel_step.